bin/cml.js

```diff
@@ -51,13 +51,29 @@ const handleError = (message, error) => {
   process.exit(1);
 };
 
-const options = {
+exports.options = {
   log: {
     type: 'string',
     description: 'Maximum log level',
     coerce: (value) => configureLogger(value) && value,
     choices: ['error', 'warn', 'info', 'debug'],
     default: 'info'
+  },
+  driver: {
+    type: 'string',
+    choices: ['github', 'gitlab', 'bitbucket'],
+    description:
+      'Forge where the repository is hosted. If not specified, it will be inferred from the environment'
```

Comment:
I'll take "unspecified" and final periods as global review suggestions.

Reply:
Yes?

- We don't use `forge` in `cml.dev`
- Everyone knows what a (web)site is

Reply:
* We don't use the term `forge` in [cml.dev](https://cml.dev) because of an historical vacancy in our team's vocabulary. After joining, I noticed that in all the meetings and conversations GitHub, GitLab and Bitbucket were being erroneously called SCMs; i.e. what Git is, but forges aren't. 🐲 

* [XKCD 1133](https://xkcd.com/1133/); where do we draw the line? Should we use overly generic terms to replace more accurate ones if we suspect than the latter are (allegedly) less known?

Reply:
- agree that `SCM` is wrong
- agree that `(web)site` is better
- would also like `Git host`
- disagree that `forge` is best. I'd personally find `forge` super confusing and wrong, it's not even jargon - none of the dictionary definitions support a techy use of the word. Note that GitHub self-describes as a "developer platform" while GitLab a "DevOps platform" and Bitbucket "Git based code hosting".

Reply:
> _agree that `(web)site` is better_

(Looking up “unilateral agreement” on the dictionary) 😄 

`(web)site` is better than `SCM` in the sense that is not plain wrong, but not better than forge to my mind.

Reply:
+1 for forge personally.

But we can just say `Forge (Git host provider) ...` to clarify, and unblock this.
Sounds ok?

Reply:
according to poll of @iterative members, it looks like `git provider` or `git host` are 4~5x more popular than `forge`

Reply:
And the question was _“What generic term for "GitHub/GitLab/Bitbucket" do you think users will understand best?”_

Reply:
**Top 5 choices of 9**

* `git provider` — 46% (16 voter)
* `git host` — 34% (12 voter)
* `forge` — 9% (3 voter)
* `git forge` — 6% (2 voter)
* `git site` — 3% (1 voter)

Reply:
Resolving _ad populum_ with 093136ad5925e155889a0e9c2e659546b8566dae